src/graphics/pyro/pyro.h

```diff
@@ -65,67 +63,47 @@ class CPyro
     friend class CPyroManager;
 
     //! Creates pyrotechnic effect
-    bool        Create(PyroType type, CObject* obj, float force);
+    bool        Create();
     //! Destroys the object
     void        DeleteObject();
```

Comment:
`DeleteObject()` should probably be called `DeleteLight()` because it never deletes any other kind of object and because a similar name may cause confusion with a completely unrelated `DeleteObject(bool primary, bool secondary)` method